tutorials.rst

```diff
@@ -102,25 +102,25 @@ As you probably realised, if you didn't know it already: the important thing in
 teach the child to cook. The only thing that really matters is that the child should enjoy the experience of working in
 the kitchen with you, and gains **confidence**, and wants to do it again.
 
-That needs to be the outcome of each learning experience in the kitchen; if it's not, then even if the child
+That needs to be the outcome of each learning experience in the kitchen; if it's not enjoyable, then even if the child
```

Comment:
It feels to me a bit redundant. Moreover, this change reduces the meaning of the mentioned _it_. It is not only about _enjoyable_, but about the entire experience of having something _enjoyable_, of having _experience_ in the kitchen and the most important _**confidence**_. The aforementioned _it_ encompasses all of this. @warmana, I suggest to omit word _enjoyable_.

Reply:
Good point, thank you. I've rephrased - hopefully it's better now.